### PR TITLE
Bug 1225047 - Convert contacts_list_overlay to use the gaia-menu web component

### PR DESCRIPTION
**Pull request by KevinGrandon**
https://bugzilla.mozilla.org/show_bug.cgi?id=1225047

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f359e59452d7f99a94f61f0ad32ea4da4104c323

File: apps/communications/dialer/test/unit/suggestion_bar_test.js [hunk 1]
```diff
@@ -405,20 +401,14 @@ suite('suggestion Bar', function() {
...
+        sinon.assert.called(LazyLoader.load);
       });
 
       test('overlay is displayed', function() {
-        assert.equal(subject.overlay.getAttribute('aria-hidden'), 'false');
+        assert.equal(subject.overlay.hasAttribute('hidden'), false);
         assert.isTrue(subject.overlay.classList.contains('display'));
       });
 
-      test('should have a cancel button as the last button', function() {
```
Comment (KevinGrandon) [inline apps/communications/dialer/test/unit/suggestion_bar_test.js]: This test is no longer needed as the cancel button is button is behind the shadowRoot, and gaia_menu has its own tests.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=bd6d27718235ec09b721c511af598026f87b317d

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f84ae52979c9402ab510b918d4d1c2fcff00d50d

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5de308ac1314fb3b084438f45a167437312a3bbf

**Pull request merged**